lib/ds/hashmap.flow

```diff
@@ -221,6 +222,16 @@ pairs2hashMapWithLoadFactor(pairs : [Pair<?, ??>], hash : (?) -> int, loadFactor
 	hashMap
 }
 
+pairs2hashMapArray(pairs : [Pair<?, [??]>], hash : (?) -> int) -> HashMap<?, [??]> {
+	len = cast(cast(length(pairs) : int -> double) / defaultHashMapLoadFactor : double -> int) * 2;
+	hashMap = makeHashMapOfVolume(hash, len, defaultHashMapLoadFactor);
+	iter(pairs, \pair -> { 
```

Comment:
Why is this not a fold? Isn't the hashmap functional?

Reply:
copy-paste from pairs2hashMap()